scripts/workflows/README.md

```diff
@@ -0,0 +1,28 @@
+## NVMe device performance testing workflow with Popper
+
+[Popper](https://github.com/getpopper/popper) is a container-native workflow execution and task automation engine for defining and executing container-native workflows in Docker, as well as other container engines. 
+More details about Popper can be found [here](https://popper.readthedocs.io/).
+
+This folder contains a `wf.yml` file that defines a Popper workflow for automatically run benchmarks for a NVMe device with two different I/O engines: Linux-native asynchronous I/O access library (libaio) engine against the [SPDK block device layer](https://spdk.io/doc/bdev.html) through the [fio](https://fio.readthedocs.io/en/latest/fio_doc.html) tool, similarly to the tests presented in the [SPDK NVMe BDEV Performance Report Release 18.04](https://ci.spdk.io/download/performance-reports/SPDK_nvme_bdev_perf_report_18.04.pdf), and plots the results.
+
+
+### Instructions:
+
+1. Install [Docker](https://docs.docker.com/get-docker/) if you don't have it already.
+
+
+2. Install Popper
+```
+curl -sSfL https://raw.githubusercontent.com/getpopper/popper/master/install.sh | sh
+```
+
+3. Clone the repository.
+```
+git clone https://github.com/spdk/spdk.git
+```
+
+4. Run the workflow.
+```
+cd spdk/
+popper run -f scripts/workflows/wf.yml -c scripts/workflows/config.yml
+```
```

Comment:
sorry, I think I didn't explain myself well in my previous review when I made a comment regarding removing a paragraph from the README. I meant only that specific paragraph; the description of each of the stages was actually quite informative. In general, a description of what are the steps contained in the workflow and what each one does would be good to have in the README.

You can take the point of view of a reader such as yourself, when you were starting to look at this project a month ago (wow, you moved really fast!), and how you you'd liked to have a README or guide like this one explain from a high-level point of view what the benchmark does, the preparation step, etc..

Also, we can show an example of how to run a single step, and also how to run a step in interactive mode. For an example see [here](https://github.com/uccross/skyhookdm-ceph-cls#dev-setup).

Reply:
oh, sorry, got it. I'm going to add it up again with more details!

Reply:
oh, another thing we need to have is a big **WARNING** sign that cautions the users to be careful when running this workflow because it's going to destroy the contents of the device. We can also show how they can make sure, for example using:

```bash
lsblk
```

so that they know that the device they're using doesn't contain a filesystems already, or that it's already partitioned.